### DISCUSSION TITLE
[Datagrip/Webstorm] Install into 32-bit or 64-bit prefix as appropriate

### DISCUSSION THREAD

**Discussion by 9numbernine9** [chocolatey-community/chocolatey-packages#1844] [Ideas]
Greetings!

I noticed recently that both the packages for Datagrip and Webstorm always seem to install into `C:\Program Files (x86)`. This seems slightly unusual to me since they are now provided by Jetbrains as 64-bit executables and I'm installing them in a 64-bit Windows 10 environment.

Would it be possible to update the `ChocolateyInstall.ps1` scripts for these packages to install into the correct locations as appropriate? It might be useful to look at Zed's script for installing IntelliJ IDEA Ultimate; this is another Jetbrains project and the script for this package appears to behave as expected (note that it's not maintained in the `chocolatey-packages` repo). 

https://github.com/ZedG2/choco-pkgs-intellijidea/blob/main/ultimate/tools/chocolateyInstall.ps1

Thanks!

**Comment by AdmiringWorm** [comment]
This must have been missed somehow, but before I can open up this issue (since it is an enhancement request).

Are you interested in working on a fix for implementing this request?
Without having someone that is willing to look into the implementation, I can not open up the issue.
